### ISSUE TITLE
Colab: training always stops at Epoch 17

### ISSUE THREAD

**Issue by Tylersuard** [junyanz/pytorch-CycleGAN-and-pix2pix#1215]
Hello.  I've done this a couple times now, and training always stops at Epoch 17.  I don't think it's a problem with the instance, because I can refresh and it's still running, and I have the colab premium with 24 hour instance time.  But I have left it to train overnight and it just stays at 17 epochs.

**Comment by yajain** [comment]
Yeah this happens to me at 87. What I did is interrupting the kernel and resuming training from the same epoch. To resume training from the same epoch use the same training line you used before but add --continue_train and epoch_count 15 to it.
It will start the training again from epoch 17 and use the latest saved model.

**Comment by EnricoBeltramo** [comment]
I have similar issue at epoch 25 with 3000 images for epoch running at my pc. There are some operation that are done periodically with that frequency?
